src/FuncSharp/DataTypes/Product/IProductExtensions.cs

```diff
@@ -7,6 +7,71 @@ namespace FuncSharp
 {
     public static class IProductExtensions
     {
+        /// <summary>
+        /// Returns hash code of the specified product.
+        /// </summary>
+        public static int ProductHashCode(this IProduct product)
```

Comment:
Not sure about these additions. Why are they here?

Reply:
Actually, if it gets generated, it means I deleted it from the generated file but not from the template. So remove it from the template and this will not be generated.